config/initializers/ahoy.rb

```diff
@@ -6,7 +6,7 @@
 module Ahoy
   class Store < Ahoy::Stores::LogStore
     def exclude?
-      return if FeatureManagement.enable_load_testing_mode?
+      return true if FeatureManagement.enable_load_testing_mode?
```

Comment:
Yeah, I was trying to figure out how to test this.  I mocked up something that I thought tested the logic on my own, and was going to test in tf, but got distracted with yesterday's terraform issue in the evening.  I probably shouldn't have made the PR until I'd done that, but I wanted to get people's eyes on it.  Apologies.

Hmm.  I guess I thought that we didn't want to emit events during load testing, and that's why we wanted to return true.  Is that not correct?